routes/index.js

```diff
@@ -0,0 +1,141 @@
+const express = require("express");
+const router = express.Router();
+const { Revise } = require("revise-sdk");
+// const { fetchGoals } = require("./data.js");
+const AUTH_TOKEN =
+  "eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpZCI6ImRiZjcxYzQyLWQyNGMtNDYxMS1hMDk3LTA4NjFmMTUwZmFjYSIsImtleSI6ImNjMzJ0OGJrIiwiaWF0IjoxNjY0ODEwMzUzfQ.f4_-DM70NODaauMACBTWBidHwf4ItZI9V10QpLuwWSk";
+const revise = new Revise({ auth: AUTH_TOKEN });
+router.get("/get-football-nft/:id", async (req, res) => {
+  try {
+    let nftId = req.params.id;
+    let nftData = await revise.fetchNFT(nftId);
+    const nft = revise.nft(nftData);
+    let nftmetaData = nft.nft.metaData;
+    return res.json({
+      message: "Successfully Retrieve Data",
+      properties: nftmetaData,
+      data: nftData,
+    });
+  } catch (error) {
+    return res.json({ message: "Server Error" });
+  }
+});
+
+router.get("/all-nfts/:collectionID", async (req, res) => {
+  try {
+    let collectionId = req.params.collectionID;
+    let nftsData = await revise.fetchNFTs(collectionId);
+
+    return res.json({
+      message: "Successfully Retrieve Data",
+
+      data: nftsData,
+    });
+  } catch (error) {
+    return res.json({ message: "Server Error" });
+  }
+});
+
+router.post("/edit-nft/:id", async (req, res) => {
+  const props = req.body;
+  try {
+    let nftId = req.params.id;
+    let previousnftData = await revise.fetchNFT(nftId);
+
+    for (let key in props) {
+      await (await revise.updateNFT(nftId))
+        .setProperty(key.toString(), props[key].toString())
+        .save();
```

Comment:
You can call save just once at the end. Instead of calling it in a loop. That way, you avoid multiple server calls.